### DISCUSSION TITLE
How do I do a zigbee2mqtt update?

### DISCUSSION THREAD

**Discussion by Trust-9** [tteck/Proxmox#2721] [Help Needed] [answered]
Hi,
I discovered this site last year and have already used some Proxmox scripts. I am really enthusiastic.
But now I want to update my Z2M container.
According to the official instructions, I should simply run update.sh.
Unfortunately, I then get an error message.
Is there a special way to update the tteck version?

`Updating ccc08a3f..ffc2ff1d
error: Your local changes to the following files would be overwritten by merge:
        update.sh
Please commit your changes or stash them before you merge.
Aborting`

Thanks

**Comment by tteck** [comment, marked as answer]
![Screenshot 2024-04-02 6 04 55 AM](https://github.com/tteck/Proxmox/assets/21342670/0e82c319-1e6b-4366-8b6d-da139bf4922e)

**Comment by Trust-9** [comment]
embarrassing ... I'm sorry
Thanks for the quick reply
